helios-client/src/main/java/com/spotify/helios/common/descriptors/DeploymentGroup.java

```diff
@@ -55,30 +53,29 @@
 public class DeploymentGroup extends Descriptor {
 
   public static final String EMPTY_NAME = "";
-  public static final Map<String, String> EMPTY_LABELS = emptyMap();
-  public static final JobId EMPTY_JOB = null;
+  public static final JobId EMPTY_JOB_ID = null;
 
   private final String name;
   private final List<HostSelector> hostSelectors;
-  private final JobId job;
+  private final JobId jobId;
   private final RolloutOptions rolloutOptions;
 
   /**
    * Create a Job.
    *
    * @param name The docker name to use.
-   * @param job The job for the deployment group.
+   * @param jobId The job ID for the deployment group.
    * @param hostSelectors The selectors that determine which agents are part of the deployment
    *                       group.
    */
   public DeploymentGroup(
       @JsonProperty("name") final String name,
       @JsonProperty("hostSelectors") final List<HostSelector> hostSelectors,
-      @JsonProperty("job") @Nullable final JobId job,
+      @JsonProperty("job") @Nullable final JobId jobId,
```

Comment:
This is just internal variable naming. The json key will still be "job" like in Deployment.java